packages/frontend/stalker-app/src/app/modules/dashboard/number-of-domains-metric/number-of-domains-metric.component.ts

```diff
@@ -11,7 +12,13 @@ import { SimpleMetric } from '../simple-metric/simple-metric.component';
   imports: [CommonModule, SimpleMetric],
 })
 export class NumberOfDomainsMetric {
-  public value$ = this.domainService.getPage(0, 1).pipe(map((x) => `${x.totalRecords}`));
+  public value$ = globalProjectFilter$.pipe(
+    switchMap((project) => {
+      const projects = [];
+      if (project) projects.push(project.id);
+      return this.domainService.getPage(0, 1, { projects }).pipe(map((x) => `${x.totalRecords}`));
```

Comment:
Food for thought, some day we will probably want a route that only counts. Page of one is fine, performance wise, but hackish